packages/fusion-runtime/src/utils.ts

```
@@ -620,3 +625,79 @@ export function millisecondsToStr(milliseconds: number): string {
+      }
+      if (wildcardTransportOptions) {
+        toBeMerged.push(wildcardTransportOptions);
+      }
+      transportEntryMap[subgraphName] = mergeDeep(toBeMerged);
+    }
+  }
+  const schemaExtensions: {
+    directives?: {
+      transport?: TransportEntry[];
+    };
+  } = (unifiedGraph.extensions ||= {});
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)